style.css

```diff
@@ -16,6 +16,10 @@ body {
   margin: 0 auto;
 }
 
+/* header {
+  position: fixed;
+} */
+
```

Comment:
- [ ] I would suggest you to kindly remove the commented code linke commented code line makes code look messy and misleading 
- [ ] Also i would suggest you kindly remove all commented code line in your readme